src/datastore_mad/remotes/lxd_downloader.sh

```diff
@@ -105,7 +105,9 @@ for p in ${arguments//&/ }; do
     [ -n "$k" -a -n "$v" ] && eval $k=$v;
 done
 
-extension="tar.xz"
+#Only works with double extensions (.tar.xz and .tar.gz)
```

Comment:
we need some error checking here, if the extension is not what we expect we should fail